payment/src/main/java/com/kyriba/conference/payment/domain/Discount.java

```diff
@@ -0,0 +1,9 @@
+package com.kyriba.conference.payment.domain;
```

Comment:
looks like a dto (move to /domain/dto ?)

Reply:
this is the domain object

Reply:
Very strange domain object. No constructor, package private fields...

Reply:
agree. more like a dto object of domain level